build_from.py

```diff
@@ -95,5 +143,13 @@ def generate_diffs(reg_tree, act_title_and_section, builder, layer_cache):
     parser.add_argument('--checkpoint', required=False,
                         help='Directory to save checkpoint data')
 
+    parser.add_argument('--operation', action='store')
```

Comment:
:+1: to a generic runner.

On five seconds worth of thought, I like `regulations-parser` though, since that's the name of the repo, and eregs is bigger than just the parser.